src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -73,9 +75,55 @@ type CA struct {
 	Cert       string // PEM-encoded
 }
 
+func exists(file string) bool {
+	if _, err := os.Stat(file); os.IsNotExist(err) {
+		return false
+	}
+	return true
+}
+
+func configExists(file string) (string, bool) {
+	configPath, err := InConfigDir(file)
+	if err != nil {
+		log.Errorf("Could not get config path? %v", err)
+		return configPath, false
+	}
+	return configPath, exists(configPath)
+}
+
+func majorVersion(version string) string {
+	return r.FindString(version)
+}
+
+// copyNewest is a one-time function for using older config files in the 2.x series.
+// from 2.0.2 forward, Lantern will consider all major versions to be compatible and
+// will name them accordingly, as in "lantern-2.yaml".
+func copyNewest(file string) {
+	// If we already have a config file with the latest name, use that one.
+	// Otherwise, copy the most recent config file available.
+	cur, exists := configExists(file)
+
+	if exists {
+		return
+	}
+	files := []string{"lantern-2.0.1.yaml", "lantern-2.0.0+stable.yaml", "lantern-2.0.0+manoto.yaml", "lantern-2.0.0-beta8.yaml"}
+
+	for _, file := range files {
+		if path, exists := configExists(file); exists {
+			if err := os.Rename(path, cur); err != nil {
+				log.Errorf("Could not rename file: %v", err)
```

Comment:
Oh yeah now with the loop that's a little easier. I did do that intentionally yeah just because we might be able to grab the next for some unknown reason.